### DISCUSSION TITLE
adding access to sponsors in Bosnia & Herzegovina

### DISCUSSION THREAD

**Discussion by Nbc66** [community/community#65701] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

adding access to sponsors in Bosnia & Herzegovina

would be a huge plus for anyone making open-source projects in Bosnia

**Comment by anjuan** [comment, marked as answer]
Thanks, @Nbc66. I'm the Engineering Manager for GitHub Sponsors, and we're regularly evaluating where to expand the Sponsors program. I'll take your suggestions back to the team. 🙂
